projects/core/directives/dropdown/dropdown-context.directive.ts

```diff
@@ -44,6 +62,48 @@ export class TuiDropdownContextDirective extends TuiRectAccessor {
         this.driver.next(false);
     }
 
+    @HostListener('touchmove.silent', [
+        '$event.touches[0].clientX',
+        '$event.touches[0].clientY',
+    ])
+    onTouchMove(x: number, y: number): void {
+        if (!this.isIOS) {
+            return;
+        }
+
+        this.currentRect = tuiPointToClientRect(x, y);
+    }
+
+    @HostListener('touchstart.silent', [
+        '$event.touches[0].clientX',
+        '$event.touches[0].clientY',
+    ])
+    onTouchStart(x: number, y: number): void {
+        if (!this.isIOS) {
+            return;
+        }
+
+        this.currentRect = tuiPointToClientRect(x, y);
+        this.zone.runOutsideAngular(() => {
```

Comment:
What's the point of running it outside if we immediately re-enter?

Reply:
why not just `runOutsideAngular`?

```ts
  this.zone.runOutsideAngular(() => {
    this.longTapTimeout = 
         setTimeout(() => this.driver.next(true)) as unknown as number;
  });
```

Reply:
fixed

Reply:
`as unknown as number;` instead of `as unknown as any;`

Reply:
It is already outside of angular because of `.silent`

Reply:
Should `touchstart` also be `.passive`?